operations/policies/policies_webhooks.rego

```diff
@@ -11,52 +11,141 @@ webhook_rule_present(obj, expected) if {
 	obj := input[_].contents
 	is_no_downscale_webhook(obj)
-    not validate_webhook(obj, "no-downscale", "/admission/no-downscale", "None", "Equivalent", rule)
-    msg := sprintf("no-downscale ValidatingWebhookConfiguration does not have expected attributes, %v", [display_name(obj)])
+    not webhook_rule_present(obj, rule)
+    msg := sprintf("ValidatingWebhookConfiguration does not have expected rule, %v. Expected %v, got %v", [display_name(obj), rule, obj.webhooks[0].rules])
+}
+
+# Assert that the pod-eviction has expected path
+deny contains msg if {
+    path := "/admission/pod-eviction"
+	obj := input[_].contents
+	is_pod_eviction_webhook(obj)
+	obj.webhooks[0].clientConfig.service.path != path
+    msg := sprintf("ValidatingWebhookConfiguration does not have expected path, %v. Expected %v, got %v", [display_name(obj), path, obj.webhooks[0].clientConfig.service.path])
 }
 
-# Assert that the pod-eviction ValidatingWebhookConfiguration has expected attributes
+# Assert that the pod-eviction has expected rules
 deny contains msg if {
     rule := { "apiGroups": [""], "apiVersions": ["v1"], "operations": ["CREATE"], "resources": ["pods/eviction"] }
 	obj := input[_].contents
 	is_pod_eviction_webhook(obj)
-    not validate_webhook(obj, "pod-eviction", "/admission/pod-eviction", "None", "", rule)
-    msg := sprintf("pod-eviction ValidatingWebhookConfiguration does not have expected attributes, %v", [display_name(obj)])
+    not webhook_rule_present(obj, rule)
+    msg := sprintf("ValidatingWebhookConfiguration does not have expected rule, %v. Expected %v, got %v", [display_name(obj), rule, obj.webhooks[0].rules])
+}
+
+
+# Assert that the zpdb-validation has expected path
+deny contains msg if {
+    path := "/admission/zpdb-validation"
+	obj := input[_].contents
+	is_zpdb_validation_webhook(obj)
+	obj.webhooks[0].clientConfig.service.path != path
+    msg := sprintf("ValidatingWebhookConfiguration does not have expected path, %v. Expected %v, got %v", [display_name(obj), path, obj.webhooks[0].clientConfig.service.path])
 }
 
-# Assert that the zpdb-validation ValidatingWebhookConfiguration has expected attributes
+# Assert that the zpdb-validation has expected rules
 deny contains msg if {
     rule := { "apiGroups": ["rollout-operator.grafana.com"], "apiVersions": ["v1"], "operations": ["CREATE", "UPDATE"], "resources": ["zoneawarepoddisruptionbudgets"] }
 	obj := input[_].contents
 	is_zpdb_validation_webhook(obj)
-    not validate_webhook(obj, "zpdb-validation", "/admission/zpdb-validation", "None", "", rule)
-    msg := sprintf("zpdb-validation ValidatingWebhookConfiguration does not have expected attributes, %v", [display_name(obj)])
+    not webhook_rule_present(obj, rule)
+    msg := sprintf("ValidatingWebhookConfiguration does not have expected rule, %v. Expected %v, got %v", [display_name(obj), rule, obj.webhooks[0].rules])
 }
```

Comment:
Are these needed? Won't the integration tests fail without these attributes set correctly now?